Sources/TuistCore/Models/Target.swift

```diff
@@ -63,7 +63,48 @@ public struct Target: Equatable, Hashable, Comparable {
                 coreDataModels: [CoreDataModel] = [],
                 actions: [TargetAction] = [],
                 environment: [String: String] = [:],
-                launchArguments: [String: Bool] = [:],
+                launchArguments: [String: Bool],
```

Comment:
Model types are internal to Tuist / and the TuistGenerator library, those have less of a need to be 100% backward compatible seeing there is a very clear / easy migration path

Reply:
✅